File: Netbeans/Java/Lab6/test/model/PerfectOrNoTest.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package model;

import org.junit.Test;
import static org.junit.Assert.*;

/**
 *
 * @author prostrmk
 */
public class PerfectOrNoTest {
    
    public PerfectOrNoTest() {
    }

    @Test
    public void testCheckerForPerfection() {
        System.out.println("checkerForPerfection");
        int choose = 496;
        boolean expResult = true;
        boolean result = PerfectOrNo.checkerForPerfection(choose);
        assertEquals(expResult, result);
        
    }
    
}

File: Csharp/Laba6/Laba6/TestFolder/PerfectOrNoTest.cs
using NUnit.Framework;
using System;

namespace Laba6
{
	[TestFixture ()]
	public class PerfectOrNoTest
	{
		[Test ()]
		public void PerfectionTest ()
		{
			int perfect = 496;
			Assert.AreEqual (true, PerfectOrNo.CheckerForPerfection (perfect));
		}
	}
}

